recipe/meta.yaml

```diff
@@ -35,7 +36,7 @@ requirements:
     - murmurhash >=0.28.0,<1.1.0
     - cymem >=2.0.2,<2.1.0
     - preshed >=3.0.2,<3.1.0
-    - thinc >=8.1.8,<8.2.0
+    - thinc >=8.1.8,<8.3.0
```

Comment:
This is going to be necessary for the python 3.12 migration (which is winding down), so it would be great to get this merged. :)